starter-code/app.js

```diff
@@ -0,0 +1,26 @@
+
+// SETUP
+const express = require("express");
+const app = express();
+app.use(express.static("public"));
+
+
+//ROUTES
+app.get("/", (request,response,next)=>{
+  response.sendFile(__dirname + "/views/home.html");
+});
+
+
+app.get("/about", (request,response,next)=>{
+  response.sendFile(__dirname + "/views/about.html");
+});
+
+
+app.get("/gallery", (request,response,next)=>{
+  response.sendFile(__dirname + "/views/gallery.html");
+});
+
+app.listen(8080, ()=> {
+  console.log("coffee time !!!")
```

Comment:
The EADDRINUSE error means that a process is already using the port your are trying to use. Usually, if port 3000 is already in use, it means that a server you created earlier in the day for another app hasn't been shut down properly. 
To forcefully shut it, you can do this : 
```
lsof -i tcp:3000
```
It would output something like following:
```
COMMAND  PID   USER   FD   TYPE  DEVICE  SIZE/OFF NODE NAME
node     5805  xyz    12u  IPv6  63135    0t0     TCP  *:3000 (LISTEN)
```
Now kill the process using :
```
kill -9 5805
```
(replace 5805 by the PID you get)
more info here : 
https://stackoverflow.com/questions/4075287/node-express-eaddrinuse-address-already-in-use-kill-server